WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsViewModel.swift

```diff
@@ -241,6 +234,22 @@ private extension WooShippingCreateLabelsViewModel {
         }
         stores.dispatch(action)
     }
+
+    /// Syncs origin addresses to use for shipping label from remote.
+    ///
+    func loadOriginAddresses() {
+        let action = WooShippingAction.loadOriginAddresses(siteID: order.siteID) { [weak self] result in
+            guard let self else { return }
+            switch result {
+            case .success(let addresses):
+                originAddresses = addresses
```

Comment:
I am wondering, do we need to keep the `originAddresses` array, or just the `selectedOriginAddress`?